src/views/Player.vue

```diff
@@ -18,20 +18,39 @@
       v-if="isQuestionShown"
       data-test="modal"
     ></QuestionModal>
+
+    <Scorecard
+      id="scorecardmodal"
+      class="absolute z-10"
+      :class="{
+        hidden: !isScorecardShown,
+      }"
+      :metrics="scorecardMetrics"
+      :progressPercentage="scorecardProgress"
+      :isShown="isScorecardShown"
+      :title="title"
+      greeting="Hooray! Congrats on completing the quiz! 🎉"
+      :numQuestionsAnswered="numQuestionsAnswered"
+      @restart-quiz="restartQuiz"
```

Comment:
upon clicking go back, we are not restarting the quiz right? We are just showing the last answered question!